packages/console/less/console.less

```diff
@@ -3,13 +3,3 @@
 */
 @import 'serial-console';
 @import 'vnc-console';
-
-.console-actions-pf {
-  float: right;
-  position: relative;
-  top: -2em;
-}
-
-.console-actions-buttons-pf {
-  margin-left: 5px;
-}
```

Comment:
These classes are used by `SerialConsole`

Reply:
In the meantime, I moved them to `serial-console` styling